src/components/deposit.js

```diff
@@ -36,12 +36,21 @@ const Deposit = props => {
 
   const handleDepositClick = () => {
     const vaultContract = props.vaultContract()
-    const amount        = toWeiFormatted(new BigNumber(deposit), props.decimals)
+    let   amount        = toWeiFormatted(new BigNumber(deposit), props.decimals)
 
     setDepositLabel('Deposit...')
     setStatus('deposit')
 
-    vaultContract.methods.deposit(amount).send({ from: props.address }).then(() => {
+    let call
+
+    // Native Matic vault
+    if (vaultContract.methods.depositMATIC) {
```

Comment:
I assume there is no way to change this contract call to match the others, but I have to ask =)

Not possible or it is just because we are in a rush?

Reply:
You can't "deposit" native tokens (matic in polygon eth in ethereum), you have to send a payable transaction... I tried... but it doesn't work =)

Reply:
Damn =(